### DISCUSSION TITLE
As a Harvesters, will I compete the proof speed with other?

### DISCUSSION THREAD

**Discussion by cccat6** [Chia-Network/chia-blockchain#1895] [Q&A] [answered]
In the architecture, I see that said it will have a constant in each proof so that I will have at least 30secs to find the proof. However, I have a storage that needs a couple of seconds to randomly access a position in the drive. Is it possible I and the other harvesters have the same proof and we compete who find it at first? If it is true, I may have no chance to win since my random access is too low.
I also wondering it says "at least 30secs", but what is "at most"?
Thank you.

**Comment by hoffmang9** [comment, marked as answer]
Your proof isn't competing with another proof and will get in if sent to the network in under 30 seconds from receiving the signage point. Any harvester lookup that takes more than 5 seconds should be avoided for optimal farming.
